README.md

```diff
@@ -72,6 +72,49 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-check/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+### What features does `cargo-semver-checks` enable in the tested crates?
```

Comment:
There's a thought exercise that I think is worth trying for this, if you're up for it.

Pretend you're a user of this tool, not a contributor.

When might you come looking for this section? What questions might you have about it?

Were you able to find the answers quickly and efficiently? If not, what changes would help with that?

Did reading the section leave you with more unanswered questions? If so, can we either rephrase to avoid introducing the confusion that raises those questions, or answer them immediately?

Ideally, the section should _succinctly_ answer the user's questions, avoid ambiguity, and leave the user with an impression of "this feature is well-designed and does what I want."

Reply:
I will try to come up with a short list of possible questions tomorrow, I am too tried already today 😵‍💫

Reply:
Putting it here for documentation's sake.

Some scenarios (put in order of how probable they seem to me, from most to least):
- You are considering adding cargo-semver-checks to your project's CI and you are curious if it would work for your usecase. 
  - does cargo-semver-checks detect API changes for every feature configuration?
  - what features are enabled in the checks?
  - how much can the feature set be customized?
- You are trying to use adapt cargo-semver-checks in your project's CI,
  - what features exactly are enabled by default?
  - how to enable a specific set of features?
  - how does a flag work?
 - Running cargo-semver-checks results in a false-positive, because of the feature configuration. 
   - is this the expected behaviour?
   - how to fix this issue?
 - You add (or possibly remove) a feature into your project that already uses cargo-semver-checks
   -  how does cargo-semver-checks interacts with that?
   - how to make sure the new feature is checked?
   - how to run checks when removing a feature?

Reply:
I looked through this list, and I think that answers to almost all the questions can be found reading the README, possibly requiring to also look at the tool's help message. 

I am not sure about the last point:
> * You add (or possibly remove) a feature into your project that already uses cargo-semver-checks
>   * how does cargo-semver-checks interacts with that?
>   * how to make sure the new feature is checked?
>   * how to run checks when removing a feature?

Reply:
Great catch! This last point is worth making a new FAQ entry for — let's do a separate PR for that so we can merge this one faster.

If you'd prefer not to make that PR, could you instead just open an issue about it so we don't forget?